src/wallet/cc_wallet/cc_wallet_puzzles.py

```diff
@@ -15,42 +16,23 @@
 from src.util.ints import uint64
 
 
-def cc_make_puzzle(innerpuzhash, core):
-    # Puzzle runs the core, but stores innerpuzhash commitment
-    puzstring = f"(r (c (q 0x{innerpuzhash}) ((c (q {core}) (a)))))"
-    result = Program.to(binutils.assemble(puzstring))
-    return result
+MOD = load_clvm("coloured_coins.clvm")
```

Comment:
I think we'll have to add this to pyinstaller spec, similar to how src/util/english.txt is added currently.